### ISSUE TITLE
VideoGame microdata example contains rdfa

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#289]
http://lists.w3.org/Archives/Public/public-vocabs/2015Jan/0108.html

Thanks, Aaron Bradley for catching this:

"""... Quick note that one of the microdata examples for schema.org/VideoGame
contains RDFa syntax.

The "Monopoly" example has this for microdata:
    ````<body vocab="http://schema.org/" itemscope itemtype="
http://schema.org/VideoGame http://schema.org/MobileApplication">````

This should simply read:
    ````<body itemscope itemtype="http://schema.org/VideoGame
http://schema.org/MobileApplication">````

"""

**Comment by Aaranged** [comment]
Another issue with this example is that there is a property mismatch between the different syntaxes.

Microdata:
<span itemprop="reviewCount">33</span> reviews

RDFa:
<span property="reviewCount">33</span> reviews

JSON-LD:
"ratingCount":"33"

The property referenced should be the same for all three examples.  (As there's not reviews present on or linked from the example code I would suggest using ratingCount for all three rather than reviewCount - but consistency here is more important then which property is used, as both are valid for AggregateRating.)

**Comment by danbri** [comment]
Thank goodness *someone* reads this stuff. Ahem.

**Comment by danbri** [comment]
Fixed in 1aa0098 and test site updated:
http://sdo-stantz.appspot.com/VideoGame#games-4
